opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractJettyTest.java

```diff
@@ -44,10 +42,29 @@
     public static final String CLIENT_ATTRIBUTE = "clientBuilder";
     public static final String TRACER_ATTRIBUTE = "tracer";
 
-    protected Server jettyServer;
-    protected MockTracer mockTracer = new MockTracer(new ThreadLocalScopeManager(), MockTracer.Propagator.TEXT_MAP);
+    // static to close it at the end
+    static Server jettyServer;
+    protected final MockTracer mockTracer = new MockTracer();
     protected final String contextPath = "/context";
-    protected Client client;
+    protected final Client client;
+
+    public AbstractJettyTest() {
+        this.client = getClient();
```

Comment:
nit: Might be better to move all this init code into a `@BeforeClass` method, so symmetry with the `@AfterClass` to cleanup?

Reply:
Subclasses override tracing config and servlet context. If I use `@BeforeClass` then it has to be defined in all subclasses, this is less intrusive.